app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt

```diff
@@ -30,12 +33,15 @@ object RateUsHelper {
       dialog.window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
       val btClose = dialog.findViewById<ImageView>(R.id.ivCross)
       val btYes = dialog.findViewById<Button>(R.id.btnSave)
+      val btLater = dialog.findViewById<Button>(R.id.btnLater)
+      val btNever = dialog.findViewById<Button>(R.id.btnNever)
       btClose.setOnClickListener {
         onCancelSelected(activity)
         dialog.dismiss()
```

Comment:
Same as backpress. I think we can keep interaction as false when back pressed or close is clicked